Gemfile

```
@@ -80,3 +81,5 @@ group :test do
 end
 
 gem 'carrierwave'
+
+gem 'rails-i18n', '~> 7.0.6'
```

```suggestion
gem 'rails-i18n'
```

特別な理由がない限りバージョンは固定しない方がいいと思います〜。
https://qiita.com/jnchito/items/823323e276d1a90ac4a1#gemfile%E3%81%A7%E3%81%AF%E3%81%AA%E3%82%8B%E3%81%B9%E3%81%8F%E3%83%90%E3%83%BC%E3%82%B8%E3%83%A7%E3%83%B3%E3%82%92%E6%8C%87%E5%AE%9A%E3%81%97%E3%81%AA%E3%81%84

---

ありがとうございます。
削除いたしました